webapp/graphite/render/functions.py

```diff
@@ -4049,6 +4050,39 @@ def randomWalkFunction(requestContext, name, step=60):
             int(epoch(requestContext["endTime"])),
             step, values)]
 
+def seriesByTag(requestContext, *tagExpressions):
+  """
+  Returns a SeriesList of all series matching the specified tag expressions
+
+  Example:
+
+  .. code-block:: none
+
+    &target=seriesByTag("tag1=value1", "tag2!=value2")
+
+  Returns a seriesList containing all matching series
+  """
+
+  if STORE.tagdb is None:
+    return []
+
+  taggedSeries = STORE.tagdb.find_series(tagExpressions)
+  if not taggedSeries:
+    return []
+
+  taggedSeriesQuery = 'group(' + (','.join(taggedSeries)) + ')'
```

Comment:
i think the brackets are not necessary: `taggedSeriesQuery = 'group(' + ','.join(taggedSeries) + ')'`